pkgs/games/prismlauncher/default.nix

```diff
@@ -46,51 +33,29 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-7tptHKWkbdxTn6VIPxXE1K3opKRiUW2zv9r6J05dcS8=";
   };
 
-  nativeBuildInputs = [ extra-cmake-modules cmake file jdk17 ninja wrapQtAppsHook ];
-  buildInputs = [
-    qtbase
-    qtsvg
-    zlib
-    quazip
-    ghc_filesystem
-    tomlplusplus
-  ]
-  ++ lib.optional (lib.versionAtLeast qtbase.version "6") qtwayland
-  ++ lib.optional gamemodeSupport gamemode.dev;
+  nativeBuildInputs = [ extra-cmake-modules cmake jdk17 ninja ];
+  buildInputs =
+    [
+      qtbase
+      zlib
+      quazip
+      ghc_filesystem
+      tomlplusplus
+    ]
+    ++ lib.optional gamemodeSupport gamemode;
+
+  hardeningEnable = [ "pie" ];
```

Comment:
Why is this suddenly required without an update to the software?

Reply:
This isn't required per-se, but I was reading up on the nixpkgs manual, and it was recommended to enable this for applications that interact with remote services, and Prism Launcher does a lot of that.

Reply:
I think that part is outdated https://github.com/NixOS/nixpkgs/blob/0a3aeb76b9cd260a9597707e9d4e8c2ee59fda97/pkgs/stdenv/generic/make-derivation.nix#L194

also you want to look at the unstable doc for contributing